app/views/home/index.html.erb

```diff
@@ -6,20 +6,21 @@
   </div>
 
 
-<div class = viewall>
+<div class = "viewall">
       
-  <div class = t-pos>
+  <div class = "t-pos">
     <table class="table table-dark table-striped table-hover ">
 
       <tr>                          
         <th>Title</th>
         <th>Author</th>
+        <th> </th>
       </tr>
       <% @posts.each do |p| %>     
       <tr class = data>
         <td id ="title"><%= p.title %></td>  
         <td id ="author"><%= p.author %></td> 
-        <td id ="blogbtn"> <%= link_to "Learn more", blog_path(blog) %></td>
+        <td id ="blogbtn"> <%= link_to "View Blog", home_path(p) %></td>
```

Comment:
In general, we want our `id` tags to be _unique_ to that element alone. Because here you are iterating with the `.each`, that means every Post title has `id="title"`, and every author has `id="author"` etc.

So if we want to style all `titles` we should probably make this a `class` instead of an `id`, as it applies to multiple elements. 

If on the other hand, we wanted to use the `id` as a _unique_ identifier for each post's title( e.g may be to find this specific title element with Javascript), we need to make it _unique_. Here we could do this by adding `.each_with_index do |p, i|` where `i` would be the index of this post in the `@posts` array. We can then interpolate that into our `id` maybe as `id="title-#{i}". This way the first post title would have `id="title-0"` and the second `id="title-1"`.

People often do `#{i+1}` so their `id`s start from 1 instead of 0 but I don't think it's too important.

In your case, you can probably remove the `id`s altogether as they aren't really doing much :)